docs/developing.md

```diff
@@ -80,27 +80,34 @@ A Kubernetes or OpenShift cluster is required to run Hive from source. If you wo
 
 [Kind](https://github.com/kubernetes-sigs/kind) can be used as a lightweight development environment for deploying and testing Hive. Currently, we support 0.8.1 version of Kind. The following instructions cover creating an insecure local registry (allowing for dramatically faster push/pull), and configuring your host OS, as well as the kind cluster to access it. This approch runs Hive in a container as you would in production, giving you the best coverage for manual testing.
 
-This approach requires [Docker](https://docs.docker.com/install).
+This approach requires either [Podman](https://podman.io/) or [Docker](https://docs.docker.com/install).
 
 If you encounter ImagePullErrors, and your kind container cannot reach your registry container, you may be experiencing problems with Fedora (at least 32, possibly earlier as well) and Docker. You can attempt to work around this by changing the FirewallBackend in the /etc/firewalld/firewalld.conf file from nftables to iptables and restarting docker. (see [this comment](https://github.com/kubernetes-sigs/kind/issues/1547#issuecomment-623756313)
 
 
 Create a local insecure registry (if one does not already exist) and then a kind cluster named 'hive' to deploy to. You can create additional clusters if desired by providing a different name argument to the script.
 
+Podman:
+```bash
+./hack/create-kind-podman-cluster.sh hive
+export KUBECONFIG=~/.kube/hive.kubeconfig
```

Comment:
Is setting `KUBECONFIG` necessary/advisable/possible for both tools? (If so, perhaps we can consolidate these samples.)

Reply:
I do think that's a good idea. (I've never used these scripts before.)

Even better would be a single script that detects whether you've got podman or docker (preferring one if you have both -- which? probably podman) and just Does The Right Thing™. I understand if that's too much scope creep for this PR though :)